### PR TITLE
Added warning to Migrate settings about develop mode.

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) adds a warning about setting develop mode when migrating data.
This replaces https://github.com/magento/devdocs/pull/3771

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/migration/migration-migrate-settings.html
- https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-settings.html

### CODE REVIEW

File: guides/v2.2/migration/migration-migrate-settings.md [hunk 1]
```diff
@@ -20,13 +20,18 @@ According to our data migration [order]({{ page.baseurl }}/migration/migration-m
 
 2. Change to the Magento `/bin` directory or make sure it is added to your system PATH.
 
+{: .bs-callout .bs-callout-info }
+Make sure Magento is deployed in default mode. Developer mode can cause validation errors in the migration tool.
```
Comment (shrielenee) [inline guides/v2.2/migration/migration-migrate-settings.md]:
```suggestion
Ensure Magento is deployed in default mode. Developer mode can cause validation errors in the migration tool.
```

File: guides/v2.2/migration/migration-migrate-settings.md [hunk 2]
```diff
@@ -43,25 +48,27 @@ The `Migration completed` message is displayed after the settings are transferre
...
-		cp settings.xml.dist settings.xml
+   ```bash
+   cp settings.xml.dist settings.xml
+   ```
 
-4. Make your changes in `settings.xml`.
+1. Make your changes in `settings.xml`.
 
-5. Specify the new name of the settings file for mapping. To do that, change the `<settings_map_file>` tag in the `path/to/config.xml` file.
+1. Specify the new name of the settings file for mapping. To do that, change the `<settings_map_file>` tag in the `path/to/config.xml` file.
```
Comment (shrielenee) [inline guides/v2.2/migration/migration-migrate-settings.md]:
```suggestion
1. To specify the new name of the settings file for mapping change the `<settings_map_file>` tag in the `path/to/config.xml` file.
```

File: guides/v2.3/migration/migration-migrate-settings.md [hunk 3]
```diff
@@ -20,13 +20,18 @@ According to our data migration [order]({{ page.baseurl }}/migration/migration-m
 
 2. Change to the Magento `/bin` directory or make sure it is added to your system PATH.
 
+{: .bs-callout .bs-callout-info }
+Make sure Magento is deployed in default mode. Developer mode can cause validation errors in the migration tool.
```
Comment (shrielenee) [inline guides/v2.3/migration/migration-migrate-settings.md]: "Ensure" again.

File: guides/v2.3/migration/migration-migrate-settings.md [hunk 4]
```diff
@@ -43,25 +48,27 @@ The `Migration completed` message is displayed after the settings are transferre
...
-		cp settings.xml.dist settings.xml
+   ```bash
+   cp settings.xml.dist settings.xml
+   ```
 
-4. Make your changes in `settings.xml`.
+1. Make your changes in `settings.xml`.
 
-5. Specify the new name of the settings file for mapping. To do that, change the `<settings_map_file>` tag in the `path/to/config.xml` file.
+1. Specify the new name of the settings file for mapping. To do that, change the `<settings_map_file>` tag in the `path/to/config.xml` file.
```
Comment (shrielenee) [inline guides/v2.3/migration/migration-migrate-settings.md]: See previous comment about this phrasing.

**Comment by dobooth** [comment]
running tests

**Pull request merged**